akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -73,6 +73,9 @@ object MediaType {
       override def isApplication = true
     }
 
+  def fontBinary(subType: String, comp: Compressibility, fileExtensions: String*): Binary =
```

Comment:
for `application`, we have `applicationBinary`, `applicationWithFixedCharset` and `applicationWithOpenCharset`. For `font`, I don't think we need to call the method `fontBinary`, since at least for now all `font` subtypes are binary, like with `video` and similar below - so I would suggest calling this method `font` instead (and perhaps moving it down to below `video`)